submissions/motunrayoIlawole/php/list-view.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+require_once("connection.php");
+
+setcookie( "customers", json_encode($result), time() + 3600, "/" );
+
+echo "<h3>List of registered customers</h3>";
+
+try {
+    $sql = "SELECT * FROM registered_customers";
+
+    $result = $pdo->query($sql);
+    $customers = $result->fetchAll();
+
+    if ($result->rowCount() > 0) {
```

Comment:
Well done @TunrayoIlawole 

But, the idea of this task is to store the result returned from this SQL query in a cookie, Each time I reload this page, I don't want to always run this SQL query but instead, I want to get it from the data stored in the cookie as long as the cookie has not expired. If the cookie is has expired then rerun the SQL query.

With your current implementation, each time I reload this page, I am always executing this SQL query.

Reply:
What you did still doesn't change anything. What you are looping through is still the $result coming from the SQL query every time